orangecontrib/educational/widgets/utils/tests/test_optimizers.py

```diff
@@ -0,0 +1,128 @@
+from orangecontrib.educational.optimizers import *
```

Comment:
Yes, you test them all. You specified them below. List all that you import here so anyone easily sees where the identifiers come from. Star imports are only suitable in package __init__s.